install/empty_data.php

```diff
@@ -4296,15 +4296,24 @@ public function getEmptyData(): array
                 'notificationtemplates_id' => '6',
                 'language' => '',
                 'subject' => '##ticket.action## ##ticket.entity##',
-                'content_text' => '##lang.ticket.entity## : ##ticket.entity##
-
-##FOREACHtickets##
-
-##lang.ticket.title## : ##ticket.title##
- ##lang.ticket.status## : ##ticket.status##
-
- ##ticket.url##
- ##ENDFOREACHtickets##',
+                'content_text' => '##lang.ticket.authors##
+##lang.ticket.title##
+##lang.ticket.priority##
+##lang.ticket.status##
+##lang.ticket.attribution##
+##lang.ticket.creationdate##
+##lang.ticket.content##
```

Comment:
As this is not rendered as a table, you should probably add corresponding label in front of each value in the foreach loop (e.g. `##lang.ticket.title## : ##ticket.title##`)